g11n-ws/docs/developer_guide_javaclient.md

```diff
@@ -0,0 +1,25 @@
+## Developer's Guide for Singleton Java Client
+
+### How to setup Intellij IDEA for development
+
+You can follow below steps to setup your development environment:
+
+1. Install JDK 1.8(or later version prior to JDK 10) and Gradle 4.10.3(or later version prior to Gradle 5)
```

Comment:
Currently support only JDk 1.8 and gradle 4.10.3(compatible version).

Reply:
fixed